go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/aclindsa/xml
+
+go 1.12
```

Comment:
Go added this automatically, but it wasn't intentional 😅 
Should we keep it?

Reply:
Why not? If go wants it there bad enough, who am I to argue?